lms/static/scripts/frontend_apps/components/FilePickerApp.tsx

```diff
@@ -56,7 +63,7 @@ function formatContentURL(content: URLContent) {
     return 'Book from VitalSource';
   }
 
-  return truncateURL(content.url, 65 /* maxLength */);
+  return truncateURL(content.url, 50 /* maxLength */);
```

Comment:
65 chars was too long! It was breaking layout in some extreme, non-wrapping cases.